Libraries/LibGUI/GItemView.h

```diff
@@ -55,4 +55,5 @@ class GItemView : public GAbstractView {
     bool m_rubber_banding { false };
     Point m_rubber_band_origin;
     Point m_rubber_band_current;
+    Vector<GModelIndex> m_rubber_band_remember_selections;
```

Comment:
Let's not use verbs in variable names. Also, let's use the singular "selection" for a set of selected indices.
`m_rubber_band_remembered_selection`?
`m_rubber_band_preserved_selection`?
`m_rubber_band_original_selection`?

Reply:
I like `m_rubber_band_remembered_selection` "remembering" something is a fun abstraction :) I'll get this patched up ASAP.